src/Core/Services/SyncService.cs

```diff
@@ -274,6 +274,66 @@ public async Task<bool> SyncDeleteCipherAsync(SyncCipherNotification notificatio
             return SyncCompleted(false);
         }
 
+        public async Task<bool> SyncUpsertSendAsync(SyncSendNotification notification, bool isEdit)
+        {
+            SyncStarted();
+            if (!await _stateService.IsAuthenticatedAsync())
+            {
+                return SyncCompleted(false);
+            }
+
+            try
+            {
+                var localSend = await _sendService.GetAsync(notification.Id);
+                if (localSend != null && localSend.RevisionDate >= notification.RevisionDate
+                    && ((isEdit && localSend == null) || (!isEdit && localSend != null)))
```

Comment:
```suggestion
                if ((localSend != null && localSend.RevisionDate >= notification.RevisionDate)
                    || (isEdit && localSend == null) || (!isEdit && localSend != null))
```

I think the if should be like this. Thank you for taking your time making this changes. I really appreciate it